src/server.c

```diff
@@ -221,13 +221,26 @@ int loop(void) {
 	}
 }
 
+void setup_signal_handlers(void) {
+	struct sigaction sa;
+
+	// don't transform child processes into zombies and don't handle SIGCHLD.
+	sigemptyset(&sa.sa_mask);
+	sa.sa_flags = SA_NOCLDSTOP | SA_NOCLDWAIT | SA_RESTART;
+	sa.sa_handler = SIG_DFL;
```

Comment:
dwm uses `SIG_IGN` here, but I think we want the default signal handlers for everything else right?

Reply:
Nice catch! This changes only what we need.